Doc/library/http.server.rst

```diff
@@ -420,47 +420,78 @@ such as using different index file names by overriding the class attribute
 
 .. _http-server-cli:
 
+Command-line interface
+----------------------
+
 :mod:`http.server` can also be invoked directly using the :option:`-m`
 switch of the interpreter.  Similar to
 the previous example, this serves files relative to the current directory::
 
-        python -m http.server
+   python -m http.server
```

Comment:
```suggestion
   python -m http.server [OPTIONS] [port]
```

Reply:
So I think we should change next sentence: `Similar to the previous example, this serves files relative to the current directory`

Reply:
Oh, yes there is no previous example :') You can say something like "The following example illustrates how to serve files relative to the current directory"

Reply:
That's what I wrote :)